.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and test ContratosPR API
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install Pipenv
+      run: pip install pipenv
+    - name: Install dependencies
+      run: pipenv install --dev
+    - name: Run Black
+      run: pipenv run black --check .
+    - name: Run iSort
+      run: pipenv run isort -q --recursive --check-only
+    - name: Run Flake8
+      run: pipenv run flake8 -q
```

Comment:
Can we try this instead?

```
    - name: Install pre-commit
       run: pip install pre-commit
     - name: Run pre-commit
       run: pre-commit run --all-files
```

Reply:
I'm not convinced about this. 

For one thing we would be mixing the use of `pipenv` and `pip` in our CI. I think that this would confuse some folks.

I also wanted to be explicit in these steps. Not using `pre-commit` like this shows whoever takes a peek exactly what we expect to pass. It also lets us get rid of `pre-commit` in the future with out having to change our CI scripts.

Thoughts?

Reply:
We could still use pipenv to install the pre-commit dev dependency(but why do we have to install absolutely everything else for this?).

> It also lets us get rid of pre-commit in the future with out having to change our CI scripts.

Same would apply to changes to our `pre-commit` hooks, no? If I want to add the following, we wouldn't be able to have parity for linting checks between dev and CI.

```
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.3.0
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: debug-statements
```

Reply:
All this is totally fair. Call me convinced.

Reply:
Followed your first suggestion. Your initial thoughts were spot on.